qiskit/result/distributions/quasi.py

```diff
@@ -48,6 +48,7 @@ def __init__(self, data, shots=None):
             ValueError: If the string format of the keys is incorrect
         """
         self.shots = shots
+        self._stddev_upper_bound = None
```

Comment:
This needs an init kwarg to set this value without manually setting an internal attribute.

Reply:
Done